indexing-service/src/main/java/io/druid/indexing/overlord/TaskQueue.java

```diff
@@ -290,7 +291,7 @@ public String apply(Task task)
         }
         // awaitNanos because management may become necessary without this condition signalling,
         // due to e.g. tasks becoming ready when other folks mess with the TaskLockbox.
-        managementMayBeNecessary.awaitNanos(60000000000L /* 60 seconds */);
+        managementMayBeNecessary.awaitNanos(TimeUnit.SECONDS.toNanos(60));
```

Comment:
Might be nice to pull this out into a constant if you're feeling motivated

Reply:
Done.